lib/ta_course_page.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+
+import 'data.dart';
+
+class TaCoursePage extends StatefulWidget {
+  final String title, taId, courseId;
+
+  TaCoursePage(this.title, this.taId, this.courseId);
+
+  @override
+  _TaCoursePageState createState() => _TaCoursePageState();
+}
+
+class _TaCoursePageState extends State<TaCoursePage> {
+  TA ta;
+  Course course;
+  TaCourse taCourse;
+
+  getData() async {
+    ta = await getTaById(widget.taId);
+    course = await getCourseById(widget.courseId);
+    taCourse = await getTaCoursePair(ta.id + '_' + course.id);
+    setState(() {});
+    print(ta.name + ' - ' + course.name);
```

Comment:
`    print(ta.name + ' - ' + course.name + ' - ' + taCourse.rating.toString());
` or delete it idk 😹

Reply:
No 😛

Reply:
toxic👀